exonum/CHANGELOG.md

```diff
@@ -85,7 +85,7 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - Changed iterators over `Patch` and `Changes` data into custom types instead
   of standard collection iterators. (#393)
 
-- Fixed typo in `SparceListIndexKeys` and `SparceListIndexValues`. (#398)
+- Fixed typo in `SparseListIndexKeys` and `SparseListIndexValues`. (#398)
```

Comment:
I don't think that you should fix _that_

Reply:
Lol, find-replace is a good tool, but not in that case :D

Reply:
How should I fix that? I can't add invalid spelling into the dictionary, obvsly. 
cc @DarkEld3r @slowli

Reply:
Can the exception be added to the changelog directly?

```markdown
# Changelog

<!-- spell-checker:ignore sparce -->
```

Alternatively,

```markdown
<!-- spell-checker:disable -->
- Fixed typo in `SparceListIndexKeys` and `SparceListIndexValues`. (#398)
<!-- spell-checker:enable -->
```

Both seem to work locally for me, but I dind't check whether these changes interfere with the Markdown linter.